README.md

```diff
@@ -41,11 +41,11 @@ Create an empty directories called `urls`, `data` and `ckan`
 mkdir urls data ckan
 ```
 
-From a terminal, use php-cli (command line interface) to run:
+From a terminal, run:
 ```
-php grab_urls.php
+python grab_urls.py
```

Comment:
Add

```
source pyenv/bin/activate
```

beforehand, if it fits in here - maybe it would sit alongside the virtual environment (as suggested above) docs better.

Reply:
Fixed in 03bf889.